recipes/protobuf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+    name: protobuf
+    version: "3.0.0a3"
```

Comment:
Wasn't aware they finally updated. I haven't tried it yet, but I don't see any reason not to.